app/controllers/admin/terms_of_service_files_controller.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+module Admin
+  class TermsOfServiceFilesController < Spree::Admin::BaseController
```

Comment:
I don't see any calls to `#authorize!` in this new controller, and none of the specs are checking permissions, so my immediate thought are:
- Is that being handled automagically via `Spree::Admin::BaseController`?
- Do we have specific CanCan permissions for this resource?
- Can any logged in user delete any Terms of Service PDFs?
- What happens if a regular enterprise user tries to access that `#delete` action?

A couple of basic specs for this controller which show these things would be good :+1:

Reply:
In fact, the controller spec has only one test on one of the four actions. It definitely needs some more coverage!

Reply:
👍 3b5991865a977b79aa87a5861cfcf19bcd9d621f